api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -543,6 +543,15 @@ message HttpConnectionManager {
   // emitted by the connection manager.
   repeated config.accesslog.v3.AccessLog access_log = 13;
 
+  // The interval to flush the above access logs. By default, the HCM will flush exactly one access log
+  // on stream close, when the HTTP request is complete. If this field is set, the HCM will flush access
+  // logs periodically at the specified interval. This is especially useful in the case of long-lived
+  // requests, such as CONNECT and Websockets. Final access logs can be detected via the
+  // `requestComplete()` method of `StreamInfo`.
+  // The interval must be at least 1 millisecond.
```

Comment:
I mentioned this privately - but I'm a bit concerned about the unexpected filter interactions. I think some filters register for `onLog` as some sort of end-of-stream signal. Wasm, for example. Once they start getting multiple calls, they might do something weird. There are  two ways I can think of:
* make filters deal with it, there are other callbacks `onStreamComplete` and `onDestroy`.
* make this setting finer grained, e.g. per access log formatter.

If we go with this change (option 1), we should warn downstream projects about a possible bad interaction.

Reply:
Yeah that's a good point. With that said, this is an opt-in feature so maybe it's ok while we sort out issues like that? I could go either way.

Reply:
Yeah, I think this is a good feature so we should bring it in. Mostly concerned about how to minimize disruption.

Reply:
I updated the comment, and the changelog description, in addition to adding a check in the WASM filter.